airflow/providers/amazon/aws/transfers/mysql_to_s3.py

```diff
@@ -60,6 +75,11 @@ class MySQLToS3Operator(BaseOperator):
     :type index: str
     :param header: whether to include header or not into the S3 file
     :type header: bool
+    :param file_format: the destination file format, only string 'csv' or 'parquet' is accepted.
+    :type file_format: str
+    :param pd_kwargs: arguments to include in pd.to_parquet or pd.to_csv.
+        This is preferred than pd_csv_kwargs.
```

Comment:
```suggestion
    :param pd_kwargs: arguments to include in ``DataFrame.to_parquet()`` or
        ``DataFrame.to_csv()``. This is preferred than ``pd_csv_kwargs``.
```

I think those underscores would mess up restructuredText rendering if they are not wrapped in backticks.

Also the docstring should mention of ``pd_csv_kwargs`` (and also mention it's deprecated), otherwise the last sentence wouldn't make sense.